### PR TITLE
Added vanity_url_code field in class guild

### PR DESCRIPTION
**Pull request by AmeyWale**
## Summary

Added vanity_url_code field in class guild

## Checklist

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by SebbyLaw** [comment]
In the future, please give your pull request a sensible title and follow the pull request format by providing a summary of what your pull request is trying to achieve.

**Comment by AmeyWale** [comment]
> In the future, please give your pull request a sensible title and follow the pull request format by providing a summary of what your pull request is trying to achieve.

I did give a summary about adding a new field of vanity_url_code in class guild but I dont really know why it is not showing in the PR

**Comment by SebbyLaw** [comment]
gitkraken_T1CR6xbMCw

It looks like you commented it out.

**Comment by AmeyWale** [comment]
> gitkraken_T1CR6xbMCw
> 
> It looks like you commented it out.

I am sorry this is my first PR , might missed out on something

**Review by Rapptz — CHANGES_REQUESTED** [review]
Hello,

Thanks for the PR. It's appreciated. Changes like these should be documented similar to other attributes under the `Attributes` header. There are many examples to follow so it should be straightforward to add.

File: discord/guild.py [hunk 1]
```diff
@@ -203,6 +203,10 @@ class Guild(Hashable):
         The guild's description.
     verification_level: :class:`VerificationLevel`
         The guild's verification level.
+    vanity_url_code: Optional[:class:`str`]
+        The guild's vanity url code, if any
+
```
Comment (Rapptz) [inline discord/guild.py:208]:
```suggestion
```

**Review by Rapptz — APPROVED** [review]

File: discord/guild.py [hunk 2]
```diff
@@ -203,6 +203,9 @@ class Guild(Hashable):
         The guild's description.
     verification_level: :class:`VerificationLevel`
         The guild's verification level.
+    vanity_url_code: Optional[:class:`str`]
+        The guild's vanity url code, if any
+        .. versionadded:: 2.0
```
Comment (Rapptz) [inline discord/guild.py:208]:
```suggestion
    vanity_url_code: Optional[:class:`str`]
        The guild's vanity url code, if any

        .. versionadded:: 2.0
```

File: discord/guild.py [hunk 3]
```diff
@@ -456,6 +460,7 @@ def _from_data(self, guild: GuildPayload) -> None:
         self.max_video_channel_users: Optional[int] = guild.get('max_video_channel_users')
         self.premium_tier: int = guild.get('premium_tier', 0)
         self.premium_subscription_count: int = guild.get('premium_subscription_count') or 0
+        self.vanity_url_code : Optional[str] = guild.get('vanity_url_code')
```
Comment (Rapptz) [inline discord/guild.py:463]:
```suggestion
        self.vanity_url_code: Optional[str] = guild.get('vanity_url_code')
```

**Review by Rapptz — CHANGES_REQUESTED** [review]
Sorry about that, these changes should appease the annoying formatter.

File: discord/guild.py (hunk 1 above)
Comment (Rapptz) [inline discord/guild.py:208]: Unfortunately, this is failing due to the trailing whitespace in this line. The suggestion I made didn't have the whitespace, though it was probably hard to tell. In the future, if you want to apply a suggestion from GItHub there's a dedicated button to do so:

image

It's pretty handy! Usually projects don't like trailing whitespace, so setting up your editor to trim whitespace on save might be a desirable setting. You can find out how to do this by googling your editor.

Sorry for the annoyance on the formatting. Black touts itself as an "uncompromising formatter", so it tends to be very fussy.

**Review by Rapptz — APPROVED** [review]
Good work, sorry about having to deal with formatting again.

**Pull request merged**